### ISSUE TITLE
Multiple nav landmarks in dev docs

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#5792]
**Describe the bug**

In our dev docs pages, we have two navbars with no label that differentiate them.

**To Reproduce**
1. Go to https://ethereum.org/en/developers/
2. See the two nav bars.
![image](https://user-images.githubusercontent.com/468158/160429341-1e2de4cd-3755-4847-b2ff-ad2acaad7f8f.png)

**Expected behavior**

Add a aria-label in each nav to differentiate them.
- First nav `<nav aria-label="Primary">...<nav>`
- Second nav `<nav aria-label="Developers docs">...<nav>`

**Additional context**

Check as a reference https://www.aditus.io/patterns/multiple-navigation-landmarks/

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by HursitTarcan** [comment]
I can fix this too

**Comment by minimalsm** [comment]
Hey @HursitTarcan,

Just noting the side nav on [our developer docs](https://ethereum.org/en/developers/docs/) is also a navigation landmark and should get a differentiating aria-label too.

<img width="1561" alt="Screenshot 2022-03-30 at 11 08 58" src="https://user-images.githubusercontent.com/62268199/160807741-ab5f47f5-e30f-49c3-87bc-f8e054e8d675.png">
